docs/date.md

```diff
@@ -71,10 +71,35 @@ Returns a new `Date` object which is the sum of the current object plus the addi
 
 Returns a new `Date` object which is the sum of the current object minus the additional argument.
 
-### date.**difference**(_other_: Temporal.Date) : Temporal.Duration
+### date.**difference**(_other_: Temporal.Date, _largestUnit_: string = 'days') : Temporal.Duration
+
+**Parameters:**
+- `largestUnit` (optional string): The largest unit of time to allow in the resulting `Temporal.Duration` object.
+  Valid values are `'years'`, `'months'`, and `'days'`.
+  The default is `days`.
 
 Returns a new `Duration` object which is the difference between the current `Date` object and the argument `Date` value.
 
+**TODO:** Rest of the documentation in another pull request, this to be rebased
+
+The `largestUnit` parameter controls how the resulting duration is expressed.
+The returned `Temporal.Duration` object will not have any nonzero fields that are larger than the unit in `largestUnit`.
+A difference of two hours will become 7200 seconds when `largestUnit` is `"seconds"`, for example.
+However, a difference of 30 seconds will still be 30 seconds even if `largestUnit` is `"hours"`.
```

Comment:
This text needs to be changed for Temporal.Date, which does not support a largest unit of "hours".